pkg/convert/pot.go

```diff
@@ -21,6 +21,8 @@ const (
 	MsgStr       poTag = "msgstr"
 )
 
+const pluralCountLimit = 2
```

Comment:
In general, keep everything as close as possible! If const is used within a single function, do not declare it at the package level. Declare it inside the function that consumes it.

There are no benefits to declaring it at the package level.

Reply:
A constant pluralCountLimit is used in two different functions. (FromPot and ToPot).